src/together/filemanager.py

```diff
@@ -260,12 +266,17 @@ def _redirect_error_handler(
                 http_status=response.status_code,
             )
 
-    def redirect_policy(
```

Comment:
The current method name was confusing, because it was not named as a verb (I actually thought it's calling some policy object, which sounds more complicated than the way things really are). Renamed for clarity